decidim/decidim
# Archive Debates

#### :tophat: What? Why?
This PR introduces the "Archive" action for debates, in order to improve the navigation experience in the debates management section for admins. It divides the debates management section into "Archived Debates" and "Active Debates". A debate can also be archived or unarchived from the close debate form.

#### :pushpin: Related Issues
- Related to [MetaDecidim proposal 15687](https://meta.decidim.org/processes/roadmap/f/122/proposals/15687)

#### Testing
*Archive from the close debate form*
- Visit the debates component for any participatory space in the admin
- Click the "Lock" button to close a debate
- Check the "Archive" checkbox
- Submit the form by clicking on "Close"
- Check that the archived debate appears on the "Archived Debates" section

*Unarchive from the close debate form*
- Visit the debates component for any participatory space in the admin
- Click on the "Archived debates" blue button at the top to see the archived debates
- Click the "Lock" button to close a debate
- Uncheck the "Archive" checkbox
- Submit the form by clicking on "Close"
- Check that the unarchived debate appears on the "Active Debates" section

*Archive from the debates list*
- Visit the debates component for any participatory space in the admin
- Click the "Folder" button to archive the debate
- Click on the "Archived debates" blue button at the top to see the archived debates

*Unarchive from the debates list*
- Visit the debates component for any participatory space in the admin
- Click on the "Archived debates" blue button at the top to see the archived debates
- Click the "Folder" button to unarchive the debate
- Click on the "Active debates" blue button at the top to see the unarchived debate

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots

![Archive Debates Overview](https://user-images.githubusercontent.com/8806781/100357426-f7273680-2ff4-11eb-94f7-d9762d8b6233.gif)

:hearts: Thank you!
